File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Grid;



namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            gridView1.OptionsView.ColumnAutoWidth = false;
            WindowState = FormWindowState.Maximized;
            gridView1.OptionsBehavior.EditorShowMode = DevExpress.Utils.EditorShowMode.MouseDownFocused;
            gridView1.OptionsSelection.MultiSelect = true;
            gridView1.OptionsSelection.MultiSelectMode = GridMultiSelectMode.CellSelect;
            gridControl1.DataSource = CreateTable(20, 20);
            new GridViewHelper(gridView1);
        }



        private DataTable CreateTable(int columnCount, int rowCount)
        {
            DataTable tbl = new DataTable();
            for (int i = 0; i < columnCount; i++)
                tbl.Columns.Add(String.Format("Column{0}", i), typeof(string));
            for (int i = 0; i < rowCount; i++)
                tbl.Rows.Add(new object[] { });
            return tbl;
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Data
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Views.Grid



Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			gridView1.OptionsView.ColumnAutoWidth = False
			WindowState = FormWindowState.Maximized
			gridView1.OptionsBehavior.EditorShowMode = DevExpress.Utils.EditorShowMode.MouseDownFocused
			gridView1.OptionsSelection.MultiSelect = True
			gridView1.OptionsSelection.MultiSelectMode = GridMultiSelectMode.CellSelect
			gridControl1.DataSource = CreateTable(20, 20)
			Dim tempVar As New GridViewHelper(gridView1)
		End Sub



		Private Function CreateTable(ByVal columnCount As Integer, ByVal rowCount As Integer) As DataTable
			Dim tbl As New DataTable()
			For i As Integer = 0 To columnCount - 1
				tbl.Columns.Add(String.Format("Column{0}", i), GetType(String))
			Next i
			For i As Integer = 0 To rowCount - 1
				tbl.Rows.Add(New Object() { })
			Next i
			Return tbl
		End Function
	End Class
End Namespace